<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B747MDQ" failures="2" tests="3" name="Test(failed)(failed)(failed)" time="20.471" errors="0" timestamp="2025-04-23T19:00:03 IST">
  <testcase classname="testCases.TC003_LoginTestDDT" name="verify_loginDDT" time="10.37"/>
  <testcase classname="testCases.TC003_LoginTestDDT" name="verify_loginDDT" time="0.125">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginTestDDT.verify_loginDDT(TC003_LoginTestDDT.java:84)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginTestDDT" name="verify_loginDDT" time="0.011">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginTestDDT.verify_loginDDT(TC003_LoginTestDDT.java:84)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test(failed)(failed)(failed) -->
